draft-ietf-cdni-ci-triggers-rfc8007bis.xml

```diff
@@ -1181,9 +1211,11 @@ uCDN-A                         dCDN-B                         dCDN-C
 
    <t hangText="Name: labels">
    <vspace blankLines="1"/>
-   Description: Array of trigger labels.
+   Description: Array of trigger labels, as described in <xref target="trigger-labels"/>.
    <vspace blankLines="1"/>
-   Value: Array of Trigger labels. Each label is encoded as a JSON string and should be unique within the array of trigger labels.
+   Value: Array of trigger labels. Each label is a key-value pair, encoded as a JSON string, with "=" seprarator. 
+   The label key and value parts MUST begin with a letter or a number, MAY contain leters, numbers, hyphens, dots 
+   and udnerscores, up to 63 characters each. 
```

Comment:
Maybe reword slightly to say:  "The label key and value parts may contain up to 63 characters each, MUST begin with a letter or number, and MAY contain ..."

Also, s/udnerscores/underscores/

Reply:
accepted, only changed to MUST contain up to 63 characters each